doc/api/errors.md

```diff
@@ -1004,6 +1004,22 @@ Used when a callback is called more then once.
 can either be fulfilled or rejected but not both at the same time. The latter
 would be possible by calling a callback more then once.
 
+<a id="ERR_NET_INVALID_LISTEN_ARG"></a>
+### ERR_NET_INVALID_LISTEN_ARG
```

Comment:
Any reason not to use the existing `ERR_INVALID_ARG_TYPE` instead of creating this new error?